lib/pure/unicode.nim

```diff
@@ -971,14 +971,14 @@ iterator split*(s: string, seps: openArray[Rune] = unicodeSpaces,
   ## And the following code:
   ##
   ## .. code-block:: nim
-  ##   for word in split("this:is;an$example", {';', ':', '$'}):
+  ##   for word in split("this:is;an$example", ";:$".toRunes):
```

Comment:
* use `runnableExamples` so it gets tested in CI
* modernize it (see recent PR's that modernize examples), eg use `suggar.collect` or `toSeq` and assert instead of relying on echo/stdout

Reply:
If I wanna see my changes to I run `nim doc` over `unicode.nim`?
(I'm sorry for the questions I'm new to all this too D:)

Reply:
yeah, you could run `nim doc lib/pure/unicode.nim` I guess?